examples/tutorials/region-arguments.py

```diff
@@ -0,0 +1,156 @@
+"""
+Set the region argument
+=======================
+
+Many of the plotting functions take an argument for the ``region`` argument, which sets
+the area that will be shown in the figure. This tutorial covers the different types of
+inputs that it can accept.
+"""
+
+import pygmt
+
+########################################################################################
+# Coordinates
+# -----------
+#
+# A string of coordinates can be passed to ``region``, in the form of
+# *xmin*\ /*xmax*\ /*ymin*\ /*ymax*\ .
+
+fig = pygmt.Figure()
+fig.coast(
+    # Sets the x-range from 10E to 20E and the y-range to 35N to 45N
+    region="10/20/35/45",
+    # Set projection to Mercator, and the figure size to 15 centimeters
+    projection="M15c",
+    # Set the color of the land to light gray
+    land="lightgray",
+    # Set the color of the water to white
+    water="white",
+    # Display the national borders and set the pen-size to 0.5p
+    borders="1/0.5p",
+    # Display the shorelines and set the pen-size to 0.5p
+    shorelines="1/0.5p",
+    # Set the frame to automatic and display gridlines
+    frame="ag",
+)
+fig.show()
+
+########################################################################################
+#
+# The coordinates coordinates can be passed to ``region`` as a list, in the form
+# of [*xmin*\ ,\ *xmax*\ ,\ *ymin*\ ,\ *ymax*\ ].
+
+fig = pygmt.Figure()
+fig.coast(
+    # Sets the x-range from 10E to 20E and the y-range to 35N to 45N
+    region=[10, 20, 35, 45],
+    projection="M12c",
+    land="lightgray",
+    water="white",
+    borders="1/0.5p",
+    shorelines="1/0.5p",
+    frame="ag",
+)
+fig.show()
+
+########################################################################################
+#
+# GMT has the option for region coordinates to be passed in classic mode, which was the
+# standard prior to GMT version 6. The string format takes the coordinates for the
+# bottom-left and top-right coordinates. To specify classic mode, append **+r** at the
+# end of the ``region`` string.
```

Comment:
I don't think the descriptions are correct. 

The _xlleft/ylleft/xuright/yuright_**+r** syntax is usually used for oblique projections. It's not related to classic mode.

https://docs.generic-mapping-tools.org/latest/cookbook/options.html#data-domain-or-map-region-the-r-option

Reply:
My mistake. Thought that was a holdover to the original GMT. I'll update it.